astropy/coordinates/tests/test_frames.py

```diff
@@ -1499,6 +1519,39 @@ def test_galactocentric_references(reset_galactocentric_defaults):
                 assert k in galcen_custom.frame_attribute_references
 
 
+def test_galactocentric_registry():
+    """Test registerration in galactocentric_frame_defaults"""
+    from astropy.coordinates import (ICRS,
```

Comment:
Import might as well be on top.

Reply:
I agree. I didn't have it up top because all the other tests `test_frames` import inside the function definition. If I move it up top, I'll should also delete the in-function imports elsewhere.

Reply:
let's leave it as is for now (a separate clean-up PR would be most welcome, though!)